_posts/2024-01-12-release-0_4_1.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title:  "Kroxylicious release 0.4.1"
+date:   2023-01-12 00:00:00 +0000
+author: "Sam Barker"
+author_url: "https://www.github.com/SamBarker"
+categories:  releases kroxylcious topic-encryption
+---
+
+The Kroxylicious project is very pleased to announce the release of [Kroxylicious 0.4.1](https://github.com/kroxylicious/kroxylicious/releases/tag/v0.4.1). This release is focused on [fixing](https://github.com/kroxylicious/kroxylicious/issues/841) an issue where the proxy failed to preserve offsets while decrypting records. While we feel a little sheepish that our own testing (which we have improved at both [unit](https://github.com/kroxylicious/kroxylicious/pull/843) and [integration](https://github.com/kroxylicious/kroxylicious/pull/839/files#diff-f6ca390d21c7e5bcd3134c97c204d8394ada9217ae791da9da1c7255c8a97a75) levels) for 0.4.0 missed it we got a great bug report from [@giacomoa](https://github.com/giacomoa). Speaking of great bug reports this release also includes our first external contribution [PR#849](https://github.com/kroxylicious/kroxylicious/pull/849) from [@luozhenyu](https://github.com/luozhenyu) which fixes an issue in the SASL authentication handling.
+
+This release also includes some other small changes cleaning up examples and updating dependencies. 
+
+The whole Kroxylicious team is very excited to see people testing out the proxy and is really looking forward to hearing more about how & where people want to use it.
+
+### Encryption at Rest
```

Comment:
Should we drop this and the binary artefacts section, they are duped from 0.4.0. Maybe you could make it shorter and mention 0.4.0 plus link to the older blog if you want to maximise what this blog is delivering.

Reply:
ah the binaries could be a standard part of the release blog right

Reply:
I was keeping it as the core feature of 0.4.x and trying to push it in front of anyone stumbling across the announcement. 

I did re-word the binaries section to just say that they are available which felt like something the post should say explicitly, though format is entirely up for grabs.

Reply:
Yeah probably right that we should make it easy to view the demo. In that case it's a bit funky how it launches into talking about Filter. Maybe needs an intro paragraph.

Reply:
like the previous blog had higher up:

The big news is this release includes our `EnvelopeEncryption` filter which provides an **Encryption-at-Rest** solution for Apache Kafka(tm) which is _transparent_ to both clients and brokers.